<ui version="4.0" >
 <class>KeyDetail</class>
 <widget class="QDialog" name="KeyDetail" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>KeyDetail</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="tlHeader" >
       <property name="font" >
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Details of the RSA key</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="image" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>95</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>95</width>
         <height>40</height>
        </size>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabel1" >
       <property name="text" >
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="keyDesc" >
       <property name="toolTip" >
        <string>The internal name of the key used by xca</string>
       </property>
       <property name="maxLength" >
        <number>32767</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="keyPubEx" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="TextLabel3" >
         <property name="text" >
          <string>Keysize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="keyLength" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="tlPrivEx" >
       <property name="text" >
        <string>Private Exponent</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="tlPubEx" >
       <property name="text" >
        <string>Public Exponent</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="ClickLabel" name="keyPrivEx" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="tlModulus" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Modulus</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="keyModulus" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="but_ok" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>151</width>
         <height>29</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>widgets/clicklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>but_ok</sender>
   <signal>clicked()</signal>
   <receiver>KeyDetail</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>201</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel" >
     <x>57</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
